relationship_server.py

```diff
@@ -13,38 +13,45 @@
 logger = logging.getLogger(__name__)
 
 
-'''
-    Utliity method to convert keys and values in a dictionary to lowercase.
-'''
-def makeDictLowercase(d):
-    lowerCaseDict = dict()
+def make_dict_lowercase(d):
+    """
+        Utliity method to convert keys and values in a dictionary to lowercase.
+    """
+    lower_case_dict = dict()
     for k in d.keys():
-        lowerCaseDict[k.lower()] = d[k].lower()
-    return lowerCaseDict
-
-'''
-    Utility to convert string to Title Case
-    Input: this is a sentence.
-    Output: This Is A Sentence.
-'''
-def toTitleCase(string):
+        lower_case_dict[k.lower()] = d[k].lower()
+    return lower_case_dict
+
+
+def to_title_case(string):
```

Comment:
```suggestion
def to_title_case(input_string):
```

`string` is a Python library

Reply:
Removed the function altogether. Used `.title()` as suggested.